docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md

```diff
@@ -0,0 +1,95 @@
+# Dasharo Security: Early Boot DMA Protection
```

Comment:
@miczyg1 we should also add this new test suite to `mkdocs` file. Without this, the test suite should not be visualized in the menu on the left of the screen.

Reply:
Done